mmv1/products/compute/BackendService.yaml

```diff
@@ -1259,16 +1259,43 @@ properties:
           ClientTlsPolicy is a resource that specifies how a client should authenticate
           connections to backends of a service. This resource itself does not affect
           configuration unless it is attached to a backend service resource.
-        required: true
+        # required: true
         custom_expand: 'templates/terraform/custom_expand/resourceref_with_validation.go.erb'
       - !ruby/object:Api::Type::Array
         name: 'subjectAltNames'
         description: |
           A list of alternate names to verify the subject identity in the certificate.
           If specified, the client will verify that the server certificate's subject
           alt name matches one of the specified values.
-        required: true
```

Comment:
What's the reason for commenting both `required`?

Reply:
Both these fields are listed in the API docs as optional, and should not be required when using awsV4Authentication that sits in the same nested object.

Reply:
confirmed that they are marked as optional in docs: https://cloud.google.com/compute/docs/reference/rest/v1/backendServices

In that case can you just remove the field entirely instead of commenting it out?